### PR TITLE
Avatar&badge

### PR DESCRIPTION
**Pull request by sahiba0915**
Added Avatar and Badge page

### CODE REVIEW

**Comment by yogesh-** [comment]
While doing a PR:
- Add image or video of the screen
- write about what items and sub-items have you developed
- Add a preview link if possible via Netlify/Vercel

**Review by yogesh- — COMMENTED** [review]
Looks good to merge

File: component/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  margin-bottom: 1.5rem;
+  font-size: 1.5rem;
+  color: var(--secondary-text);
+}
+
+.badge-title {
+  font-size: 2rem;
+  padding-top: 1.5rem;
+  margin-bottom: 1.5rem;
+  color: black;
```
Comment (sharmayatendra) [inline component/badge/badge.css]: try to avoid using hard coded color

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  font-weight: 300;
+  padding-top: 1rem;
+  line-height: 3rem;
+  color: var(--secondary-text);
+}
+
+/* Sidebar */
+
+.main-content {
+  margin: 50px auto;
```
Comment (sharmayatendra) [inline styles.css:155]: you can use relative units if a value is more than 10px.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+.side-nav {
+  margin-left: 1.5rem;
+}
+
+.doc-content {
+  grid-column: 2/5;
+}
+
+.sidebar-heading {
+  color: #000000;
```
Comment (sharmayatendra) [inline styles.css:171]: you declared this in root as variable try to use that variable

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+}
+
+.sidebar-list li a {
+  text-decoration: none;
+  font-size: 1.5rem;
+  color: var(--secondary-text);
+}
+
+.sidebar-list li a:hover {
+  color: #000000;
```
Comment (sharmayatendra) [inline styles.css:190]: same as above

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+.footer .link:hover {
+  color: var(--secondary-color);
+}
+
+/* media-queries */
+
+@media only screen and (max-width: 480px) {
+  .list-item-inline {
+    display: block;
+    margin: 1rem 0rem 1rem 0rem;
```
Comment (sharmayatendra) [inline styles.css:253]: you can still use more shorthand like "margin: 1rem 0" it will automatically sets same for left & bottom

**Review by sharmayatendra — COMMENTED** [review]
you can look at the suggested changes. otherwise good to go

File: component/badge/badge.css (hunk 1 above)
Comment (sahiba0915) [reply component/badge/badge.css]: done

File: styles.css (hunk 2 above)
Comment (sahiba0915) [reply styles.css:155]: done

File: styles.css (hunk 3 above)
Comment (sahiba0915) [reply styles.css:171]: done

File: styles.css (hunk 4 above)
Comment (sahiba0915) [reply styles.css:190]: done

File: styles.css (hunk 5 above)
Comment (sahiba0915) [reply styles.css:253]: done

**Comment by sahiba0915** [comment]
Merging avatar&badge branch.

**Pull request merged**